Documentation/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-storage.md

```diff
@@ -220,6 +220,135 @@ If the Rook cluster has more than one filesystem and the application pod is sche
 
 Once you have pushed an image to the registry (see the [instructions](https://github.com/kubernetes/kubernetes/tree/release-1.9/cluster/addons/registry) to expose and use the kube-registry), verify that kube-registry is using the filesystem that was configured above by mounting the shared filesystem in the toolbox pod. See the [Direct Filesystem](../../Troubleshooting/direct-tools.md#shared-filesystem-tools) topic for more details.
 
+## Consume the Shared Filesystem across namespaces
+
+A PVC that you create using the `rook-cephfs` storage class can be shared between different Pods simultaneously, either read-write or read-only, but is restricted to a single namespace (a PVC is a namespace-scoped resource, so you cannot use it in another one).
+
+However there are some use cases where you want to share the content from a CephFS-based PVC among different Pods in different namespaces, for a shared library for example, or a collaboration workspace between applications running in different namespaces.
+
+You can do that using the following recipe.
+
+* Create a PVC in a namespace, for example:
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: base-pvc
+  namespace: first-namespace
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 100Gi
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
+
+* The corresponding PV that is created will have all the necessary info to connect to the CephFS volume (all non-necessary information are removed here):
+
+```yaml
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: pvc-a02dd277-cb26-4c1e-9434-478ebc321e22
+  annotations:
+    pv.kubernetes.io/provisioned-by: rook.cephfs.csi.ceph.com
+  finalizers:
+    - kubernetes.io/pv-protection
+spec:
+  capacity:
+    storage: 100Gi
+  csi:
+    driver: rook.cephfs.csi.ceph.com
+    volumeHandle: >-
+      0001-0011-rook-0000000000000001-8a528de0-e274-11ec-b069-0a580a800213
+    volumeAttributes:
+      clusterID: rook
+      fsName: rook-cephfilesystem
+      storage.kubernetes.io/csiProvisionerIdentity: 1654174264855-8081-rook.cephfs.csi.ceph.com
+      subvolumeName: csi-vol-8a528de0-e274-11ec-b069-0a580a800213
+      subvolumePath: >-
+        /volumes/csi/csi-vol-8a528de0-e274-11ec-b069-0a580a800213/da98fb83-fff3-485a-a0a9-57c227cb67ec
+    nodeStageSecretRef:
+      name: rook-csi-cephfs-node
+      namespace: rook
+    controllerExpandSecretRef:
+      name: rook-csi-cephfs-provisioner
+      namespace: rook
+  accessModes:
+    - ReadWriteMany
+  claimRef:
+    kind: PersistentVolumeClaim
+    namespace: first-namespace
+    name: base-pvc
+    apiVersion: v1
+    resourceVersion: '49728'
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
+
+* On this PV, change the `persistentVolumeReclaimPolicy` parameter to `Retain` to avoid it from being deleted when you will delete PVCs.
```

Comment:
add one more step to change it back any the last PVC to be deleted to cleanup the subvolume or else it needs manual cleanup.